examples/undocumented/python/graphical/lda.py

```diff
@@ -1,32 +1,31 @@
-from pylab import figure,pcolor,scatter,contour,colorbar,show,subplot,plot,connect
+import matplotlib.pyplot as plt
```

Comment:
I would put changes to python graphical example in a different PR than the C++ code below.

Also, note that graphical python examples are pretty much the bottom of our priority list, so let's not do more updates here for now

Reply:
I made these changes for vigsterkr can run this example, because there, and I guessed in some other places, is also the problem with `const std::shared_ptr<DenseFeatures<float64_t>>&`. I mean I did this, not because of the priorities. But now I have understood that it is a SWIG problem.

Reply:
sure thing. the patch is welcome, let's get it in, just wanted to make sure you are on the same page as us

Reply:
and it does make sense to improve the whole listing and make it ready for the new api, so then we don't have to touch it again anytime soon.